sdk/src/client/api/block_builder/input_selection/utxo_chains.rs

```diff
@@ -58,7 +58,7 @@ pub(crate) async fn get_alias_and_nft_outputs_recursively(
             match unlock_address {
                 Address::Alias(address) => {
                     let input_id = client.alias_output_id(*address.alias_id()).await?;
-                    let input = client.get_output(&input_id).await?;
+                    let input = client.get_output_with_metadata(&input_id).await?;
```

Comment:
is this now better named `input_with_meta`?

Reply:
Tbh I won't bother, Thoralf is removing this file in his PR.

Reply:
No and this code gets removed anyways in https://github.com/iotaledger/iota-sdk/pull/674